### PR TITLE
MWPW-136993 Add sitemap and query index

### PR DESCRIPTION
**Pull request by Brandon32**
* Add Sitemap and update Query Index

Resolves: [MWPW-136993](https://jira.corp.adobe.com/browse/MWPW-136993)

**Test URLs:**
- N/A

### CODE REVIEW

File: helix-query.yaml [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
-      - '/**'
+      - '/blog/'
+      - '/blog/actualites/**'
+      - '/blog/basics/**'
+      - '/blog/fragments/**'
+      - '/blog/grundlagen/**'
+      - '/blog/how-to/**'
+      - '/blog/neues/**'
+      - '/blog/perspectives/**'
+      - '/blog/perspektiven/**'
```
Comment (meganthecoder) [inline helix-query.yaml]: Why are there French and German folder names included in the US index?

File: helix-query.yaml [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
 
 indices:
-  default:
+  blog-us:
     include:
-      - '/**'
+      - '/blog/'
+      - '/blog/actualites/**'
+      - '/blog/basics/**'
+      - '/blog/fragments/**'
```
Comment (meganthecoder) [inline helix-query.yaml]: Don't we want to exclude fragments?

File: helix-query.yaml [hunk 3]
```diff
@@ -58,3 +64,129 @@ indices:
...
+      - '/uk/blog/basics/**'
+      - '/uk/blog/fragments/**'
+      - '/uk/blog/grundlagen/**'
+      - '/uk/blog/how-to/**'
+      - '/uk/blog/neues/**'
+      - '/uk/blog/perspectives/**'
+      - '/uk/blog/perspektiven/**'
+      - '/uk/blog/the-latest/**'
+      - '/uk/blog/topic/**'
+      - '/uk/blog/trends/**'
```
Comment (meganthecoder) [inline helix-query.yaml]: Looks like these should be `/jp/blog/`

**Review by meganthecoder — CHANGES_REQUESTED** [review]

File: helix-query.yaml (hunk 1 above)
Comment (Brandon32) [reply helix-query.yaml]: Fixed

File: helix-query.yaml (hunk 2 above)
Comment (Brandon32) [reply helix-query.yaml]: Yes, removed

File: helix-query.yaml (hunk 3 above)
Comment (Brandon32) [reply helix-query.yaml]: Fixed

**Review by meganthecoder — APPROVED** [review]
Updates look good! Is there already a sitemap-index or are you going to add it later? Also, did you link to the wrong ticket? It goes to the due diligence

**Review by JasonHowellSlavin — APPROVED** [review]

**Pull request merged**